vars/managerPipeline.groovy

```diff
@@ -0,0 +1,265 @@
+#!groovy
+
+def call(Map pipelineParams) {
```

Comment:
@fruch Isn't there a way to "inherit" from the base pipeline and only add the parameters we need for this scenario? I'd hate to think that every time we someone changes/fixes/enhances the regular longevity pipeline someone will need to remember to modify this one as well

Reply:
Maybe there is a way, but we'll need to refactor the way it's currently built. We didn't found an easy and quick way to achieve it.

We'll have to live with it, untill we'll have time to invest in such refactoring

Reply:
Oh well...